rush.json

```diff
@@ -34,6 +34,19 @@
     "url": "https://github.com/iTwin/itwinjs-core"
   },
   "projects": [
+    {
+      "packageName": "@itwin/core-extension",
```

Comment:
could you add a github codeowner rule for this pkg and the test one, and assign it caleb/bill, and myabe myself?

Reply:
Added all three of you to the extension package and added Bill (Caleb was already added) to the test folder